### ISSUE TITLE
Cannot get this to work with Airspy R2

### ISSUE THREAD

**Issue by MemeCommando** [eried/Research#1]
Hi there:

So i can connect my Airspy R2 and it works great with SDRSharp and everything. However, when i go to use your version of Tempest SDR, it does not detect the antenna and when i click load EXTIo Source it says "No HackRF devices found" followed by 'The EXtio Driver failed to initialize a device. Make sure your device is plugged in and its drivers are installed correctly. 

I am pretty sure that mine is working properly, because it works fine with SDRSharp.

Any help would be appreciated.

Thanks.

**Comment by eried** [comment]
Ah, yes this is only for the HackRF. You should open the exe file with WinRAR and add your own EXT_IO :) (it is a normal SFX file)

**Comment by phantomdennis** [comment]
@MemeCommando 

Have you looked on the Manufacture's download page it has a section for R2 Ext_IO

https://airspy.com/download/
_ExtIO for Airspy R0, R2 and Mini
This build is provided by Andrea Montefusco and works with HDSDR, Studio1 and compatible software._

**Comment by MemeCommando** [comment]
Yes, i have the EXTIO file. It just wouldnt let me load it. But i will try erieds suggestion. I also got the other version to work after hours of troubleshooting with the various Java versions.

**Comment by phantomdennis** [comment]
@MemeCommando

I have not used this in a while, but if I recall correctly you can only have one Ext_IO file in the directory at a time. If you have more than one then I vaguely recall having a problem. I kept all Ext_IO files that are not the one I'm using in a separate directory until I swap hardware or want to use a different Ext_IO
